MoTrPAC/motrpac-frontend
# Connects to #87. Animal PASS1B-06 Phenotype Data Visualization.

### Key Changes

1. Redesigned the entry pages of **Animal** and **Human** analyses.
2. Removed extraneous 2nd level UI (sub-analysis) for **Animal** Phenotype Data.
3. Implemented new boxplots for animal PASS1B-06 phenotype data provided by DMAQC.
4. Updated unit testing assertions due to UI changes.
5. Updated Storybook component due to UI changes.
6. Updated Circle CI node.js docker image tag to `10.14.2-browsers` due to failed dependency installation.

### Steps to test

1. Sign into local instance of the app.
2. Navigate to `^/analysis/animal` or `^/analysis/human` in the browser. Expect to see new layout of the main analysis types.
3. For animal analysis, expect to see only the **Phenotype Data** _card_ being active by mouseover it, while the rest of the _cards_ are inactive.
4. Click on the active animal phenotype data _card_ and expect to see the subsequent view with 4 boxplots.
5. For human analysis, expect to see only the **Published Data Meta-Analysis** _card_ being active by mouseover it, while the rest of the _cards_ are inactive.
6. Click on the active human **Published Data Meta-Analysis** _card_ and expect to see the subsequent view with 2 _cards_ as the 2nd level sub-analysis (and with the **Meta-Analysis of Public Data** _card_ being the only active one).
